src/newrelic_telemetry_sdk/client.py

```diff
@@ -119,25 +126,22 @@ def __init__(self, license_key, host=None, port=443):
                 "Content-Type": "application/json",
             }
         )
-        retries = urllib3.Retry(
-            total=False, connect=None, read=None, redirect=0, status=None
-        )
+        retries = urllib3.Retry(total=False, connect=None, read=None, redirect=0, status=None)
 
         # Check if https traffic should be proxied and pass the proxy
         # information to the connectionpool
 
         proxies = getproxies()
```

Comment:
Should we abstract this proxy logic into a helper method so `__init__` isn't as long?